testsuite/MDAnalysisTests/core/test_accumulate.py

```diff
@@ -149,3 +149,120 @@ def test_total_mass_duplicates(self, group):
         with pytest.warns(DuplicateWarning) as w:
             assert_almost_equal(group2.total_mass(), ref)
             assert len(w) == 1
+
+class TestMultipole(object):
+    """Tests the functionality of *Group.*_moment() like dipole_moment
+    and quadrupole_moment.
+    """
+    @pytest.fixture(params=levels)
+    def group(self, request):
+        u = mda.Universe(PSF, DCD)
+        return getattr(u, request.param)
+
+    # Dipole
+    def test_dipole_moment(self, group):
+        assert_almost_equal(
+            group.dipole_moment(unwrap=False), 
+            np.array([-33.9281028,  -5.1098802,  -6.4160602])
```

Comment:
Just a question how were these values generated, both here and for the tests below.

Reply:
Well, I'm lazy, so I confirmed that these functions work for water and a peptide using difference options as shown in the related "issue" and then I used the files already referenced in this test_*py file and gave the test what it wants instead of figuring out how to add a new file. I will add fringe cases as requested, however I'd like to leave these to check the `compound!="group" cases`